framework/src/modules/http_api/schema/swagger.yml

```
@@ -70,23 +70,23 @@ basePath: /api
 # Tags for organizing operations
 tags:
   - name: Accounts
-    description: Account related API calls
+    description: Account related API calls.
```

Let's change all to endpoints. 

```suggestion
    description: Account related API endpoints.
```